websites/app/src/utils/uploadFormDataToIPFS.ts

```diff
@@ -1,22 +1,25 @@
 import { fetch } from 'cross-fetch'
 
-export function uploadFormDataToIPFS(
+export async function uploadFormDataToIPFS(
   formData: FormData,
-  operation: string = 'evidence'
+  operation: string = 'evidence',
+  pinToGraph = false
 ): Promise<Response> {
-  const baseUrl = 'https://kleros-scout-app.netlify.app'
-  const url = `${baseUrl}/.netlify/functions/uploadToIPFS?dapp=curate&key=scout&operation=${operation}`
+  console.log(process.env.REACT_APP_COURT_FUNCTIONS_URL)
 
-  return fetch(url, {
+  const url = `${process.env.REACT_APP_COURT_FUNCTIONS_URL}/.netlify/functions/upload-to-ipfs?operation=${operation}&pinToGraph=${pinToGraph}`
```

Comment:
we cannot use env variables in this repo I believe. so we might need to hardcode const `REACT_APP_COURT_FUNCTIONS_URL = https://kleros-api.netlify.app` in this file

Reply:
For this purpose, i prefer just hardcoding it inside the url itself, as i don't see this api being used anywhere else besides this function